src/styles/table.scss

```diff
@@ -137,6 +140,39 @@ $block: #{$fd-namespace}-table;
 
   /****** Elements ******/
 
+  &__empty {
+    &-table-message {
+      padding: 1rem;
+      text-align: center;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      align-content: center;
+      width: 100%;
+    }
+
+    &:not(.fixed-height) .fd-table__row:last-of-type {
+      .fd-table__cell {
```

Comment:
You can't use `.fd-tabele__something`, you need to use `.#{$block}__something`.

Reply:
done